e2e/sast_test.go

```diff
@@ -45,9 +45,9 @@ var _ = Describe("E2E TEST:"+checks.CheckSAST, func() {
 			expected := scut.TestReturn{
 				Error:         nil,
 				Score:         10,
-				NumberOfWarn:  1,
+				NumberOfWarn:  2,
```

Comment:
out of curiosity, what's the increase in Warn from?

Reply:
Removed the added logging statement.